sdk/eventgrid/Azure.Messaging.EventGrid/EventGridSourceGenerator/EventGridSourceGenerator.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.9.0" PrivateAssets="all" />
```

Comment:
aren't we picking up default targets at all?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/Directory.Build.props#L8-L9

This is why I think